packages/twenty-front/src/modules/sign-in-background-mock/constants/SignInBackgroundMockCompanies.ts

```diff
@@ -889,13 +889,13 @@ export const SIGN_IN_BACKGROUND_MOCK_COMPANIES = [
     id: 'a7bc68d5-f79e-40dd-bd06-c36e6abb4678',
     domainName: 'samsung.com',
     updatedAt: '2023-11-23T15:38:03.699Z',
-    employees: null,
+    employees: 400_000,
```

Comment:
400_000 is not a valid number should be 400000

Reply:
<img width="209" alt="image" src="https://github.com/twentyhq/twenty/assets/29927851/37c47dca-6864-4717-859a-95fa27ce7d76">

Reply:
<img width="182" alt="image" src="https://github.com/twentyhq/twenty/assets/29927851/2b8e0d05-6e5f-4f27-a36f-22481be51ebe">